src/globus_sdk/services/transfer/client.py

```diff
@@ -1226,6 +1228,11 @@ def operation_ls(
         :param show_hidden: Show hidden files (names beginning in dot).
             Defaults to true.
         :type show_hidden: bool, optional
+        :param limit: Limit the number of results returned. Defaults to 100,000 ,
```

Comment:
I wasn't sure if it would impact the rendering, but I thought this is slightly easier to read because it puts some separation between the "literal" and adjacent punctuation. I don't care much, but unless it causes a rendering issue, I'd _tend_ to keep the space.